dockernet/scripts/airdrop/airdrop1.sh

```diff
@@ -1,16 +1,18 @@
-### AIRDROP TESTING FLOW
+#!/bin/bash
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 source ${SCRIPT_DIR}/../../config.sh
 
-# CLEANUP if running tests twice, clear out and re-fund accounts
-$STRIDE_MAIN_CMD keys delete airdrop-test -y &> /dev/null || true 
-$STRIDE_MAIN_CMD keys delete distributor-test -y &> /dev/null || true 
-$OSMO_MAIN_CMD keys delete host-address-test -y &> /dev/null || true 
+### AIRDROP TESTING FLOW Pt 1
+
+# This script tests airdrop claiming directly on Stride
+# This covers the case where the host zone has a coin type of 118 
+# and autopilot is disabled for the claim
 
 # First, start the network with `make start-docker`
-# Then, run this script with `bash dockernet/scripts/airdrop.sh`
+# Then, run this script with `bash dockernet/scripts/airdrop/airdrop1.sh`
```

Comment:
good catch, and generally huge thank you to you and @riley-stride for improving these tests!